sysutils/pushbroom/Portfile

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                pushbroom
+version             1.0.0
+revision            0
+categories          sysutils
+
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {gpanders.com:greg @gpanders} openmaintainer
+
+description         Sweep your filesystem clear of clutter
+long_description    ${name} is a command line utility that removes old files \
+                    from given file paths. Old files can either be deleted \
+                    or moved into another directory (such as a trash directory).
+
+homepage            https://sr.ht/~gpanders/pushbroom
+
+checksums           rmd160  c31601f66566cf6472289aaeea928cbee0f53931 \
+                    sha256  ec1fa9542f1b3cb1157bc4dadca3cf61ce1a35f4d1b1aa69949083a4f627ae36 \
+                    size    8224
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+python.default_version \
+                    39
```

Comment:
please set the `python.default_version` before specifying dependencies that use `${python.version}`; in this case i it works because the default in the python PG is also `39`.